core/src/main/scala/besom/internal/BesomModule.scala

```diff
@@ -3,40 +3,62 @@ package besom.internal
 import com.google.protobuf.struct.*
 import besom.internal.logging.{LocalBesomLogger => logger, BesomLogger}
 
+/** An abstract effect Besom module, which can be implemented for different effect types.
+  * @tparam Eff
+  *   the effect type
+  * @see
+  *   [[besom.Pulumi]]
+  * @see
+  *   [[besom.internal.BesomModule]]
+  * @see
+  *   [[besom.internal.BesomSyntax]]
+  */
 trait EffectBesomModule extends BesomSyntax:
   type Eff[+A]
 
   protected lazy val rt: Runtime[Eff]
 
+  /** Run a Pulumi program in the Besom context.
+    * @param program
+    *   the program to run
+    */
   def run(program: Context ?=> Output[Exports]): Unit =
     val everything: Result[Unit] = Result.scoped {
       for
-        _              <- BesomLogger.setupLogger()
-        runInfo        <- RunInfo.fromEnv
-        _              <- logger.debug(s"Besom starting up in ${if runInfo.dryRun then "dry run" else "live"} mode.")
-        taskTracker    <- TaskTracker()
-        monitor        <- Result.resource(Monitor(runInfo.monitorAddress))(_.close())
-        engine         <- Result.resource(Engine(runInfo.engineAddress))(_.close())
-        _              <- logger.debug(s"Established connections to monitor and engine, spawning streaming pulumi logger.")
-        logger         <- Result.resource(BesomLogger(engine, taskTracker))(_.close())
-        config         <- Config.forProject(runInfo.project)
+        _           <- BesomLogger.setupLogger()
```

Comment:
don't know why `scalafmt` decided to change this

Reply:
maxColumns=120, should be 140, set maxColumns=140, reformat it back to how it was, don't check in change in .scalafmt.conf into this commit, just revert the reformatting here, we have to fix this in one go everywhere but let's not break our blame needlessly because we'll mark the reformatting commit as blame-transparent for git later